test_flickr.py

```diff
@@ -1,13 +1,20 @@
 import pytest
 
 from flickr import Flickr
+from unittest import mock
+
+
+def get_photos_list_correct_mock_request(*args, **kwargs):
```

Comment:
I think eliminating this function and just creating and configuring mock directly in the test would help readability - it is only 2 lines.